bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/model/EventData.kt

```diff
@@ -9,29 +10,33 @@ sealed class EventData {
   @JsonClass(generateAdapter = true)
   data class GoalEvent(
     val timestamp: Long,
-    val goal_id: String,
-    val user_id: String,
+    val goalId: String,
+    val userId: String,
     val value: Double,
     val user: User,
     val tag: String,
-    val source_id: SourceID,
-    val sdk_version: String? = null,
+    val sourceId: SourceID,
+    val sdkVersion: String? = null,
     val metadata: Map<String, String>? = null,
+    @Json(name = "@type")
+    val protobufType: String? = "type.googleapis.com/bucketeer.event.client.GoalEvent",
```

Comment:
protobufType seems confusing, as we consume JSON api.

edit: never mind, I see it's coming from iOS implementation. let's keep consistency here.